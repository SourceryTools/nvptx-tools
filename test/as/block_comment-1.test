Test that a block comment following a 'K_dotted' directive isn't mis-handled in 'parse_line_nosemi'.

RUN: %target_as_cmd --no-verify %S/block_comment-1.s -o %t
RUN: cmp %S/block_comment-1.o.golden %t
