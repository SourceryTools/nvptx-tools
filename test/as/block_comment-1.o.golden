.file f1
// comment1 
.file f2
// comment2
//   line 2 
